tool-dbinspector/build.gradle

```diff
@@ -14,22 +14,22 @@ android {
     defaultConfig {
         minSdkVersion buildConfig.minSdk
         targetSdkVersion buildConfig.targetSdk
-        versionCode releaseConfig.versionCode
-        versionName releaseConfig.version
     }
 
     buildTypes {
         debug {
-            debuggable true
             minifyEnabled false
         }
         release {
-            debuggable false
             minifyEnabled true
             proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.txt"
         }
     }
 
+    compileOptions {
+        coreLibraryDesugaringEnabled true
+    }
```

Comment:
I see you added desugaring to each module.
I might be wrong, but it is not only needed for modules that depend on modules that use desugaring (sentinel module, sample app module and timber module?)?

Reply:
I will take a look at this

Reply:
Yeah best to recheck if you can but compiler was complaining about desugaring not being enabled in every module.